lib/rack/response.rb

```diff
@@ -95,11 +95,23 @@ def chunked?
       CHUNKED == get_header(TRANSFER_ENCODING)
     end
 
+    def enumerable_body?
```

Comment:
Since this is a public method, it should have it's own specs.  Otherwise, it should be a private method.